diff --git a/node_modules/vite/dist/node/chunks/dep-71eb12cb.js b/node_modules/vite/dist/node/chunks/dep-71eb12cb.js
index e4ef6ae..ff13668 100644
--- a/node_modules/vite/dist/node/chunks/dep-71eb12cb.js
+++ b/node_modules/vite/dist/node/chunks/dep-71eb12cb.js
@@ -13720,11 +13720,6 @@ function terserPlugin(config) {
                 !outputOptions.__vite_force_terser__) {
                 return null;
             }
-            // Do not minify ES lib output since that would remove pure annotations
-            // and break tree-shaking.
-            if (config.build.lib && outputOptions.format === 'es') {
-                return null;
-            }
             // Lazy load worker.
             worker || (worker = makeWorker());
             const terserPath = loadTerserPath(config.root);
